streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBStoreTest.java

```diff
@@ -338,22 +333,42 @@ public void setConfig(final String storeName, final Options options, final Map<S
         return entries;
     }
 
-    private static class ConfigurableProcessorContext extends MockProcessorContext {
-        final Map<String, Object> configs;
-
-        ConfigurableProcessorContext(final File stateDir,
-                                     final Serde<?> keySerde,
-                                     final Serde<?> valSerde,
-                                     final RecordCollector collector,
-                                     final ThreadCache cache,
-                                     final Map<String, Object> configs) {
-            super(stateDir, keySerde, valSerde, collector, cache);
-            this.configs = configs;
+    private static class ConfigurableProcessorContext extends AbstractProcessorContext {
```

Comment:
Why not stay with `MockProcessorContext` -- this should avoid to overwrite all the empty methods below?

Reply:
It's been a while since I did this, but I think it was because I needed to specify the `StreamsConfig`.

Reply:
I think we can do it in a follow-up PR after merging this one.